spec/features/categories/add_category_to_article_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature "user can add a category to an article", %q(
+
+As a user
+I want to be able to add a category to an article
+So that I can see articles on similar topics
+
+Acceptance criteria
+[x] A user can add a category when creating an article
+[x] A user can see the category on the article's page
+) do
+
+  scenario "a user can add a category to an article" do
+
+    user = FactoryGirl.create(:user)
+    article = FactoryGirl.create(:article)
+
+    sign_in_as(user)
+    visit new_article_path
+
+    fill_in 'Name', with: article.name
+    fill_in 'Url', with: article.url
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.